dev/validate_bom__deploy.py

```diff
@@ -148,6 +148,13 @@ def deploy(self, config_script, files_to_upload):
     script = []
     self.add_install_hal_script_statements(script)
     self.add_platform_deploy_script_statements(script)
+
+    # Add the version first to avoid warnings or facilitate checks
+    # with the configuration commands
+    script.append('hal --color=false config version edit'
```

Comment:
If this is a general reporting cleanup I would recommend replacing `--color=false` with `-q --log=all`